modules/aws_k8s_base/aws_k8s_base.py

```diff
@@ -74,7 +79,9 @@ def process(self, module_idx: int) -> None:
 
     def pre_hook(self, module_idx: int) -> None:
         set_kube_config(self.layer)
+        add_linkerd_label_to_kubesystem()
         list_namespaces()
```

Comment:
```suggestion
        list_namespaces()
        add_linkerd_label_to_kubesystem()
```

We are using `list_namespaces` to verify connectivity to the cluster, so we shouldn't try adding the label to the namespace until we are sure we can connect. Ditto for later instances.